velox/functions/prestosql/tests/BinaryFunctionsTest.cpp

```diff
@@ -461,6 +466,54 @@ TEST_F(BinaryFunctionsTest, fromBase64Url) {
   EXPECT_THROW(fromBase64Url("YQ=/"), VeloxUserError);
 }
 
+TEST_F(BinaryFunctionsTest, toBase32) {
+  const auto toBase32 = [&](std::optional<std::string> value) {
+    return evaluateOnce<std::string>("to_base32(cast(c0 as varbinary))", value);
+  };
+
+  EXPECT_EQ(std::nullopt, toBase32(std::nullopt));
```

Comment:
Add some test cases with `digits` and digits mixed with alphabets.

Reply:
added new test case